oremda/clients/__init__.py

```diff
@@ -18,3 +19,14 @@ def Client(type: Union[ContainerType, str] = ContainerType.Docker) -> ClientBase
         raise Exception(f'Unknown container type: {type}')
 
     return container_types[type]()
+
+
+def client_from_image(image_path: Union[str, PurePath]) -> ClientBase:
```

Comment:
Encoding the type of pipeline runner through the image name, would make pipelines created in a docker environment not shareable with someone running a singularity environment

Reply:
True, we might not use this function in the pipeline code. But I think it is still useful for scripts such as validating whether an image has the right labels or not.